build_system/src/build.rs

```diff
@@ -9,6 +9,7 @@ use std::path::Path;
 struct BuildArg {
     flags: Vec<String>,
     config_info: ConfigInfo,
+    sys_root: bool,
```

Comment:
I would prefer this spelling:
```suggestion
    sysroot: bool,
```